### PR TITLE
added business data

### LINKED ISSUE #8: UI: Overlay of Clustered Business Sites

### PR DESCRIPTION
**Pull request by MartinYTSo**

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -9,14 +9,169 @@ library(lubridate)
 # Load data
 cams <- read.csv("data/surrey_desc.csv")
 cams_data <- read.csv("data/surrey_data.csv")
+bizs<-read.csv('data/all_businessess.csv')%>%
```
Comment (gouruiyu) [inline app.R:13]: I'd suggest factor out the code related to business data into a separate R script, including from line 12 to line 134, and all the biz-related icons. In this way, it is easier to maintain for later dev work on app.R file. You can include the external R script using `source()` function

**Comment by nancywan1004** [comment]
Comments:
1. Add a marker popup with the name of the location, and more info could be added later on

**Review by nancywan1004 — COMMENTED** [review]
Comments:
1. Add a marker popup with the name of the location, and more info could be added later on

**Review by nancywan1004 — APPROVED** [review]

**Review by gouruiyu — COMMENTED** [review]
When you click on a layer toggle, the checkbox is gone instead of checked, is this intended ?
image

**Comment by MartinYTSo** [comment]
image
Its not showing the same on my end

**Comment by gouruiyu** [comment]
> image
> Its not showing the same on my end

Hmm, weird. Could be a system or library version thing. Are you using Windows? @nancywan1004 Does the same issue show up on your env?

**Comment by MartinYTSo** [comment]
Yes I am using Windows 10

**Comment by gouruiyu** [comment]
k let's approve and merge it for now. I will spin off a separate issue on investigating this since it's most likely a system UI stuff, won't affect too much.

**Pull request merged**